website/content/docs/concepts/policies.mdx

```diff
@@ -800,32 +800,36 @@ authenticated user.
+| [sys/leases/revoke-force/:prefix](/vault/api-docs/system/leases#revoke-force)                                                                          | POST              | Revoke all secrets or tokens ignoring backend errors                                                                                                |
+| [sys/leases/revoke-prefix/:prefix](/vault/api-docs/system/leases#revoke-prefix)                                                                        | POST              | Revoke all secrets generated under a given prefix                                                                                                   |
+| [sys/plugins/catalog/:type/:name](/vault/api-docs/system/plugins-catalog#register-plugin)                                                              | GET, POST, DELETE | Register a new plugin, or read/remove an existing plugin                                                                                            |
 | [sys/raw](/vault/api-docs/system/raw#list-raw)                                                                                                         | LIST, GET         | Returns a list of keys for a given path prefix                                                                                                      |
-| [sys/replication/reindex](/vault/api-docs/system/replication#reindex-replication)                                                                          | POST              | Reindex the local data storage                                                                                                                      |
+| [sys/remount](/vault/api-docs/system/remount)                                                                                                          | POST              | Moves an already-mounted backend to a new mount point                                                                                               |
+| [sys/replication/reindex](/vault/api-docs/system/replication#reindex-replication)                                                                      | POST              | Reindex the local data storage                                                                                                                      |
 | [sys/replication/performance/primary/secondary-token](/vault/api-docs/system/replication/replication-performance#generate-performance-secondary-token) | POST              | Generate a performance secondary activation token                                                                                                   |
 | [sys/replication/dr/primary/secondary-token](/vault/api-docs/system/replication/replication-dr#generate-dr-secondary-token)                            | POST              | Generate a DR secondary activation token                                                                                                            |
```

Comment:
I no longer work for a company with a Vault Enterprise license, but judging from the declarative definition in the OSS repository, there are more root-protected replication paths than just these three:

https://github.com/hashicorp/vault/blob/35e2c1665f009183088387532e17d02ded312e18/vault/logical_system.go#L104-L109

Reply:
I will open a separate ticket to review this with engineering.